### PR TITLE
[CORE-1479] Add Google Analytics to editor

### PR DESCRIPTION
**Pull request by Aapzu**

### CODE REVIEW

File: grails-app/views/layouts/_tracking.gsp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<g:if env="production">
```
Comment (Aapzu) [inline grails-app/views/layouts/_tracking.gsp]: This `if` clause needs to be tested

Comment (hpihkala) [reply grails-app/views/layouts/_tracking.gsp]: @Aapzu did you test it?

File: grails-app/views/layouts/_layoutHead.gsp [hunk 2]
```diff
@@ -50,7 +50,7 @@
 			};
 		</r:script>
 	</sec:ifLoggedIn>
-	
-	<r:layoutResources/>
+
     <g:layoutHead />
+	<r:layoutResources/>
```
Comment (hpihkala) [inline grails-app/views/layouts/_layoutHead.gsp]: Hmm, why was this line moved?

Comment (hpihkala) [reply grails-app/views/layouts/_layoutHead.gsp]: Also indent looks funky

File: grails-app/views/layouts/_tracking.gsp (hunk 1 above)
Comment (Aapzu) [reply grails-app/views/layouts/_tracking.gsp]: Decided with @hpihkala that it's not worth it to test it. Let's fix it if it doesn't work in production

**Comment by Aapzu** [comment]
@hpihkala is this ready to be merged?
